src/openshift/cluster.ts

```diff
@@ -127,19 +128,27 @@ export class Cluster extends OpenShiftItem {
             .catch((error) => Promise.reject(`Failed to login to cluster '${clusterURL}' with '${error}'!`));
     }
 
+    static async copyToClipboard(url: string, token: string) {
+        return await copyPaste.copy({clusterURL: url, ocToken: token});
```

Comment:
@sudhirverma Why are we copying the url and token here. The purpose here is that the user need not have the token and can still login to the cluster based on the oc cli parameters.